docs/source/gke.md

```diff
@@ -154,6 +154,11 @@ In order for the example to work you need to modify the cluster definition in th
 sed -i "s/<gcp_region>/${GCP_REGION}/g;s/<gcp_zone>/${GCP_ZONE}/g" examples/gke/cluster.yaml
 ```
 
+#### Mac users
+```
+sed -i '' "s/<gcp_region>/${GCP_REGION}/g;s/<gcp_zone>/${GCP_ZONE}/g" examples/gke/cluster.yaml
```

Comment:
Does `sed -i -e "s/<gcp_region>/${GCP_REGION}/g;s/<gcp_zone>/${GCP_ZONE}/g" examples/gke/cluster.yaml"` work on Mac?

Reply:
yes

Reply:
thx, then we can just update the existing one to work on both platforms

Reply:
ok, I will update this

Reply:
> sed -i -e "s/<gcp_region>/${GCP_REGION}/g;s/<gcp_zone>/${GCP_ZONE}/g" examples/gke/cluster.yaml"

Will not work on Mac. It will create a new `cluster.yaml-e` and also have too many quotes.

You can use:
`sed -i '' 's/<gcp_region>/${GCP_REGION}/g;s/<gcp_zone>/${GCP_ZONE}/g' examples/gke/cluster.yaml`
or
`sed -i '' -e 's/<gcp_region>/${GCP_REGION}/g;s/<gcp_zone>/${GCP_ZONE}/g' examples/gke/cluster.yaml`

Additional information:
https://stackoverflow.com/questions/4247068/sed-command-with-i-option-failing-on-mac-but-works-on-linux/4247319#4247319

Reply:
thanks, not sure how the ending quote got into my answer but you need double quotes to substitute the variables - thansk for the `-i` difference explanation